dist/images/ovn-config.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# run on master to configure ovn-kubernetes
+# The /etc/openvswitch/ovn_k8s.conf and /etc/sysconfig/ovn-kubernetes
+# are used by ovnkube to direct processing.
+# This script creates files that can be copied to the nodes
+
```

Comment:
When does this script get run?  Shouldn't this information already be in the configmap?

Reply:
The script is a one-time executed script, run through ansible, during the installation that sets up the configmap that is used in the daemonsets.